StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -210,7 +208,10 @@ class EmbeddedPaymentMethodsView: UIView {
                                      accessoryType: RowButton.RightAccessoryButton.AccessoryType?) {
         guard let previousSavedPaymentMethodButton = self.savedPaymentMethodButton,
               let viewIndex = stackView.arrangedSubviews.firstIndex(of: previousSavedPaymentMethodButton) else {
-            stpAssertionFailure("Function supports only when there is an existing savedPaymentMethodButton")
+            stpAssertionFailure("""
+            Function supports only when there is an existing savedPaymentMethodButton because this function is only used
+            for updating the view after selecting from a larger list of saved payment methods or updating a saved payment method.
```

Comment:
"This function should never be called when there isn't already a saved PM row because there's no way for Embedded to _add_ a saved payment method today; you can only update or remove them" - something like this makes sense to me!